containers/podman-desktop
# feat: display Kubernetes pods

Signed-off-by: Jeff MAURY <jmaury@redhat.com>

### What does this PR do?

### Screenshot/screencast of this PR

#### Kubernetes pods are now reported in the Pods list window:

![image](https://user-images.githubusercontent.com/695993/216968384-35830aa4-9206-4fc1-821d-7737f23d4c06.png)

#### Clicking on the pod goes to the pod log:

![image](https://user-images.githubusercontent.com/695993/216969495-c77e71bb-27f2-49a4-88a2-ca4394e9fd5d.png)

#### Summary for the pod is similar to the Podman pod:

![image](https://user-images.githubusercontent.com/695993/216969723-2de2a135-54e3-46e7-8645-b80264a2e66c.png)

#### Inspect for the pod gives the Kubernetes YAML pod resource:

![image](https://user-images.githubusercontent.com/695993/216970070-b8e43b6e-c177-4aca-8afa-4703a51dc2be.png)

#### The Kube tab is similar to the Inspect tab and should probaby be hidden for Kubernetes pods in the future:

![image](https://user-images.githubusercontent.com/695993/216970259-85495eb6-948f-4a3e-bcbe-65aa8c0888a4.png)




### What issues does this PR fix or reference?

Fixes #1263 

### How to test this PR?

1. Start a Kubernetes cluster (Kind, minikube,..)
2. Start one or several pods on this cluster
3. Go to Podman Desktop Pods UI
4. Ensure the pods from Kubernetes are listed there
